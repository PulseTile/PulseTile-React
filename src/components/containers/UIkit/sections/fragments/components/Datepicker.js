import React from 'react';
import DateInput from '../../../../../form-fields/DateInput';

/**
 * This component returns Datepicker section for COMPONENTS
 */
const Datepicker = () => {
    return (
        <div id="datepicker" className="ui-section">
            <strong className="ui-title">Date Picker</strong>
            <div className="form-group-wrapper">
                <div className="row">
                    <div className="col-xs-12 col-sm-6">
                        <div className="form-group form-group-sm">
                            <DateInput
                                label='Select Data'
                                input={{value: 1490227200000}}
                                meta={{
                                    active: true,
                                    error: ''
                                }}
                            />
                        </div>
                    </div>
                    <div className="col-xs-12 col-sm-6">
                        <div className="form-group form-group-sm">
                            <DateInput
                                label='Select Data'
                                input={{value: 1490227200000, disabled: true}}
                                meta={{
                                    active: true,
                                    error: ''
                                }}
                            />
                        </div>
                    </div>
                </div>
            </div>
        </div>
    );
};

export default Datepicker;
