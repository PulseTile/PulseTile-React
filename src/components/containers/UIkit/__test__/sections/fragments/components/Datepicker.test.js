import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-15';
import Datepicker from '../../../../sections/fragments/components/Datepicker';

Enzyme.configure({ adapter: new Adapter() });

describe('Component <Datepicker />', () => {
  it('should renders Datepicker with props correctly', () => {
    const component = shallow(<Datepicker />);
    expect(component).toMatchSnapshot();
  });
});
