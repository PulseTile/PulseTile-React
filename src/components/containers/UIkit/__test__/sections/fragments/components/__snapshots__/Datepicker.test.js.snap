// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component <Datepicker /> should renders Datepicker with props correctly 1`] = `
<div
  className="ui-section"
  id="datepicker"
>
  <strong
    className="ui-title"
  >
    Date Picker
  </strong>
  <div
    className="form-group-wrapper"
  >
    <div
      className="row"
    >
      <div
        className="col-xs-12 col-sm-6"
      >
        <div
          className="form-group form-group-sm"
        >
          <DateInput
            input={
              Object {
                "value": 1490227200000,
              }
            }
            label="Select Data"
            meta={
              Object {
                "active": true,
                "error": "",
              }
            }
          />
        </div>
      </div>
      <div
        className="col-xs-12 col-sm-6"
      >
        <div
          className="form-group form-group-sm"
        >
          <DateInput
            input={
              Object {
                "disabled": true,
                "value": 1490227200000,
              }
            }
            label="Select Data"
            meta={
              Object {
                "active": true,
                "error": "",
              }
            }
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
